src/components/document/sort-work-view.tsx

```diff
@@ -0,0 +1,65 @@
+import { observer } from "mobx-react";
+import { useStores } from "../../hooks/use-stores";
+import React, { useState } from "react";
+import { SortWorkHeader } from "../navigation/sort-work-header";
+import { ICustomDropdownItem } from "../../clue/components/custom-select";
+import { DecoratedDocumentThumbnailItem } from "../thumbnail/decorated-document-thumbnail-item";
+import { getDocumentContext } from "../../models/document/document";
+import { DocumentContextReact } from "./document-context";
+import { DEBUG_SORT_WORK } from "../../lib/debug";
+
+import "../thumbnail/document-type-collection.sass";
+import "./sort-work-view.scss";
+
+
+export const SortWorkView:React.FC = observer(function SortWorkView(){
+  const sortOptions = ["Group", "Student"];
+  const stores = useStores();
+  const [sortBy, setSortBy] = useState("Group");
+
+  const sortByOptions: ICustomDropdownItem[] = sortOptions.map((option)=>({
+    text: option,
+    onClick: () => setSortBy(option)
+  }));
+
+  const renderDebugView = (docs: any) => {
+    return docs.map((doc:any, idx: number) => {
+      const ct = idx + 1;
+      return (
+        <pre key={idx} style={{margin:"0px", padding: "0px", fontSize: "10px"}}>
+          {ct < 10 && " "}{ct} | {doc.key} | {doc.type} | {doc.uid}
+        </pre>
+      );
+    });
+  };
+
+  return (
+    <div key="sort-work-view" className="sort-work-view">
+      <SortWorkHeader sortBy={sortBy} sortByOptions={sortByOptions} />
+      <div className="documents-panel">
+        <div className={"tab-panel-documents-section"}>
+          { DEBUG_SORT_WORK && renderDebugView(stores.documents.all)}
+          <div className={"list"}>
+            {
+              stores.documents.all.map((doc:any, idx: number) => {
```

Comment:
This should be filtered to just the problem, personal, and learningLog docs. There won't be any learningLogs for other users in the document store, but this way we'll be ready for them when we add the new listeners for that.